src/modals/Contribute.tsx

```diff
@@ -196,7 +196,7 @@ const Contribute = () => {
                         placeholder={placeholder}
                         rules={{ required: true }}
                         style={{ fontSize: '1.5rem' }}
-                        value={contribution > 0 ? contribution : ''}
+                        value={contribution >= 0 ? contribution : ''}
```

Comment:
And for the same reason as before i can't do this as well, because i'm passing a -1 value so contribution will be defined and the input will output the -1 value instead of the placeholder.

Reply:
This code was updated.